datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -100,7 +100,9 @@ def serialize_data(data):
     # Using base64 ensures:
     # 1. Printing to stdout won't fail
     # 2. Easy parsing since there are no spaces
-    return urlsafe_b64encode(data.encode('utf-8')).decode('utf-8')
+    #
+    # TODO: Remove str() when we drop Python 2
+    return str(urlsafe_b64encode(data.encode('utf-8')).decode('utf-8'))
```

Comment:
Should we use something like this instead: `to_string(urlsafe_b64encode(ensure_bytes(data))` ?

It will also help cleaning up when we remove python2, since we will search for all occurrences of to_string, ensure_bytes, etc.

Reply:
@AlexandreYang We can't do that in `datadog_checks_dev` though as `datadog_checks_dev[cli]` does not depends on `datadog_checks_base` so `to_string` is not available. The pipeline will fail in the init steps `ddev config set core .`